sdk/storage/azure-storage-internal-avro/src/main/java/com/azure/storage/internal/avro/implementation/AvroObject.java

```diff
@@ -35,8 +42,8 @@ public long getBlockOffset() {
     /**
      * @return The index of the object in the block.
      */
-    public long getObjectBlockIndex() {
-        return objectBlockIndex;
+    public long getEventIndex() {
```

Comment:
I'd go back to `ObjectIndex`. Avro doesn't know it's used in CF context. It's supposed to be general purpose parser.

Reply:
done